src/util.ts

```diff
@@ -293,6 +293,10 @@ export function titlecase(s: string) {
   return s.charAt(0).toUpperCase() + s.substr(1);
 }
 
+function escapePatchAccess(string: string) {
+  return replaceAll(replaceAll(replaceAll(string, ']', '\\]'), '[', '\\['), '.', '\\.');
```

Comment:
Minor: This isn't very readable

Reply:
What do you suggest? Make a function to replace multiple characters or just a regex?

Reply:
a function? (or ignore this comment for now). 

I'm more interested to understand what's going on.

Reply:
The regex is not that difficult and we use regexes in other places for these kinds of things.